TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/automation/AutomationChecker.java

```diff
@@ -0,0 +1,35 @@
+package com.novoda.tpbot.automation;
+
+import com.novoda.support.ColonStringSplitter;
+
+public class AutomationChecker {
+
+    private final String serviceName;
+    private final AndroidAccessibilitySettingsRetriever accessibilitySettingsRetriever;
+    private final ColonStringSplitter colonStringSplitter;
+
+    public AutomationChecker(AndroidAccessibilitySettingsRetriever accessibilitySettingsRetriever,
+                             ColonStringSplitter colonStringSplitter,
+                             String serviceName) {
+        this.serviceName = serviceName;
+        this.accessibilitySettingsRetriever = accessibilitySettingsRetriever;
+        this.colonStringSplitter = colonStringSplitter;
+    }
+
+    public boolean isHangoutJoinerAutomationServiceEnabled() {
+        String accessibilitySettings = accessibilitySettingsRetriever.retrieveEnabledAccessibilityServices();
+        String[] accessibilityServices = colonStringSplitter.split(accessibilitySettings);
+
+        for (String service : accessibilityServices) {
+            if (matchesAutomationService(service)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private boolean matchesAutomationService(String serviceToCheck) {
```

Comment:
this class is generic enough to check whether any service (provided you know the name) is enabled or not. Suggest renaming the class or not passing the serviceName in.